README.md

```diff
@@ -30,9 +30,9 @@ Please consult the [stable documentation](https://juliatesting.github.io/Aqua.jl
 You can add the following line in README.md to include Aqua.jl badge:
 
 ```markdown
-[![Aqua QA](https://raw.githubusercontent.com/JuliaTesting/Aqua.jl/master/badge.svg)](https://github.com/JuliaTesting/Aqua.jl)
+[![Aqua QA](https://https://juliatesting.github.io/Aqua.jl/stable/assets/badge.svg)](https://github.com/JuliaTesting/Aqua.jl)
```

Comment:
don't you mean
```suggestion
[![Aqua QA](https://juliatesting.github.io/Aqua.jl/stable/assets/badge.svg)](https://github.com/JuliaTesting/Aqua.jl)
```

Reply:
In fact https://juliatesting.github.io/Aqua.jl/stable/assets/badge.svg gives a 404 error...